src/mbgl/gl/types.hpp

```diff
@@ -15,7 +15,7 @@ using VertexArrayID = uint32_t;
 using FramebufferID = uint32_t;
 using RenderbufferID = uint32_t;
 
-using AttributeLocation = int32_t;
+using AttributeLocation = uint32_t;
```

Comment:
`glGetAttribLocation` returns signed integers. I know we aren't using that call, so a comment explaining the discrepancy would be useful.